app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -1,10 +1,180 @@
+            android:contentDescription="Sign Up Image"
+            android:scaleType="fitXY"
+            android:src="@drawable/sign_up_image" />
+    </androidx.cardview.widget.CardView>
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/bottom_cardview"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@+id/top_cardView"
+        android:layout_marginTop="-30dp"
+        app:cardCornerRadius="20dp">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="#F3F4F6">
+
+            <TextView
+                android:id="@+id/tvSignUp"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="5dp"
+                android:fontFamily="@font/epilogue_bold"
+                android:text="@string/sign_up"
+                android:textColor="#379AE6"
+                android:textSize="24sp"
+                app:layout_constraintBottom_toTopOf="@+id/tvCreateAccount"
+                app:layout_constraintStart_toStartOf="@+id/tvCreateAccount" />
+
+            <TextView
+                android:id="@+id/tvCreateAccount"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="25dp"
+                android:fontFamily="@font/epilogue_semibold"
+                android:text="@string/create_an_account"
+                android:textColor="#9B9B9B"
+                android:textSize="16sp"
+                app:layout_constraintBottom_toTopOf="@+id/tvName"
+                app:layout_constraintStart_toStartOf="@+id/tvName" />
+
+            <TextView
+                android:id="@+id/tvName"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="10dp"
+                android:text="@string/name"
+                android:textColor="#323842"
+                android:textSize="15sp"
+                app:layout_constraintBottom_toTopOf="@+id/etName"
+                app:layout_constraintStart_toStartOf="@+id/etName" />
+
+            <EditText
+                android:id="@+id/etName"
+                android:layout_width="269dp"
+                android:layout_height="35dp"
+                android:background="@drawable/custom_edittext"
+                android:elevation="5dp"
+                android:textColor="@color/black"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.306" />
```

Comment:
Do not give predefined height and width, instead use margins to change the height and width. Also, remove the constraint Vertical bias and constrain the top to the bottom of tvName and add sufficient margins. Do the same for all other text fields as well.